src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -3253,6 +3462,23 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
         return !isValidMrnaProfile && hasNegativeNumbers;
     }
 
+    private isGeneProfiled(
+        gene: string,
+        coverageInformation?: CoverageInformation
+    ) {
+        if (coverageInformation) {
+            return Object.values(coverageInformation.samples).some(
+                s => !_.isEmpty(s.allGenes) || !!s.byGene[gene]
+            );
+        }
+        if (this.props.coverageInformation.isComplete) {
+            return Object.values(
+                this.props.coverageInformation.result!.samples
+            ).some(s => !_.isEmpty(s.allGenes) || !!s.byGene[gene]);
+        }
```

Comment:
something like this would be better I think

```suggestion
        const samples = coverageInformation?.samples || this.props.coverageInformation.result?.samples;
        if (samples) {
            return Object.values(samples).some(
                s => !_.isEmpty(s.allGenes) || !!s.byGene[gene]
            );
        }
```